### PR TITLE
Add support for _fseeki64 (MSC).

### PR DESCRIPTION
**Pull request by gulikoza**
Add support for _fseeki64 (MSVC, mingw64). Should fix some problems in #69 as well.

Let's see if this works in AppVeyor :-)

### CODE REVIEW

File: src/trace.h [hunk 1]
```diff
@@ -48,6 +48,9 @@
 #if defined(HAVE_FSEEKO64) && defined(WIN32)
 #  define PRINTF_CAST_U64(x) ((off64_t) (x))
 #  define PRINTF_FORMAT_U64 "%I64u"
+#elif defined(HAVE_FSEEKI64) && defined(WIN32)
+#  define PRINTF_CAST_U64(x) ((__int64) (x))
```
Comment (sourcefrog) [inline src/trace.h]: Can you ever have fseeko64 (at line 49) and not fseeki?

Comment (gulikoza) [inline src/trace.h]: Mingw (32-bit) on my system has fseeko64 but not _fseeki:

-- Looking for fseeko
-- Looking for fseeko - not found
-- Looking for fseeko64
-- Looking for fseeko64 - found
-- Looking for _fseeki64
-- Looking for _fseeki64 - not found

File: CMakeLists.txt [hunk 2]
```diff
@@ -81,6 +81,7 @@ include ( CheckFunctionExists )
 check_function_exists ( alloca HAVE_ALLOCA )
 check_function_exists ( fseeko HAVE_FSEEKO )
 check_function_exists ( fseeko64 HAVE_FSEEKO64 )
+check_function_exists ( _fseeki64 HAVE_FSEEKI64 )
```
Comment (sourcefrog) [inline CMakeLists.txt]: Maybe this should be `HAVE__FSEEKI64` to make the underscore prefix clear?

**Comment by dbaarda** [comment]
Everything in this Pull request is now included in #111, along with file compatibility related changes from other pull requests and a few tidyups. I'm going to close this in favor of #111 now.

**Pull request closed without merging**